packages/cli/src/commands/migrate/tasks/config-lint.ts

```diff
@@ -1,14 +1,25 @@
-import { resolve } from 'path';
+import { resolve, extname } from 'path';
 import { ESLint } from 'eslint';
 import { outputFileSync } from 'fs-extra';
 import { cosmiconfigSync } from 'cosmiconfig';
 import { gitAddIfInRepo, getEsLintConfigPath, determineProjectName } from '@rehearsal/utils';
-import defaultConfig from '../../../configs/default-eslint';
+import { stringify as yamlStringify } from 'yaml';
+import defaultConfig from '../../../configs/eslint-default';
 import type { ListrTask } from 'listr2';
 import type { MigrateCommandContext, MigrateCommandOptions } from '../../../types';
 
-const REHEARSAL_CONFIG_FILENAME = '.rehearsal-eslintrc.js';
-const REHEARSAL_CONFIG_RELATIVE_PATH = `./${REHEARSAL_CONFIG_FILENAME}`;
+enum REHEARSAL_CONFIG_FILENAMES {
+  JS = '.rehearsal-eslintrc.js',
+  YAML = '.rehearsal-eslintrc.yml',
+  JSON = '.rehearsal-eslintrc.json',
+}
+
+enum FORMAT {
```

Comment:
We just need to handle the legacy (deprecated but we will likely still find it) `.eslintrc` (note this is not js/yaml/json extension)

Reply:
@lynchbomb Just to clarify, we need to handle .eslintrc, .eslintrc.js, .eslintrc.json, .eslintrc.yaml, .eslintrc.yml. Right?

Reply:
Correct. `.eslintrc` is older - but we should handle it